avail-js/examples/node-examples/src/tx-balance-transfer.ts

```diff
@@ -20,10 +20,22 @@ const main = async () => {
 
     const api = await initialize(config.endpoint)
     const keyring = getKeyringFromSeed(config.seed)
-    const options = { app_id: 0, nonce: -1 }
     const decimals = getDecimals(api)
     const amount = formatNumberToBalance(config.amount, decimals)
 
+    const registry = new TypeRegistry();
```

Comment:
Instead of changing a existing example it could be beneficial to add a new one. Otherwise people are going to be scared away from the amount of code that they "need" to write.